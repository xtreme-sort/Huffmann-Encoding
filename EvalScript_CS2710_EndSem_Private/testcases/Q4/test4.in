GenerateLengthLimitedHuffman
veyjowikygtogqbhzryprpplvhavmqxfqabnkybvhzhlaopwwhlhkzsguafdatgubatorottpqmbdnlflnbnnmvbrhgezjoxgwjmfiooujbfhpzyynvgpdoiwkuqaslqsrcbmbciuadovcdpgcsyyryxjiegixyvpfgldifonedampvepkciqslqrjcpcknofhvryjlrgaownldkolhpovjijnqpbaixfsovefyjgdaxzqtudbzsivbpafjbeugykdmrsraowucqlgqlvnrgqdcbwllduhunindlxmzqrjuebhxztiudhhvsbhgdajywkrreegddegioqcombnczndeexybeeooourtaptgaswhtswvjxxtkjgsjfkhmwsmljrxnoqnkrjtsluuwxqcrqewjifarishgqznkkoldpusgljndzjxdlymfpcxfjkdsppnccacqlnkmwgexfvpsvrxmrslhmllnxeqleiplgapnueuidsifrqwobefcpupmpqqjkwpjvoirrkxnnpahfdcylwmknizyffxbiwgpqmmqqlznnjnnokmbulnortxbidcniherloopdhxjolxcckfdfxijpqujsoiewlwjfqkxmtzhmcovwqglxsngkwragxdifmgnbfyabgqtfuaoybevgwosphxlvmqpcezolpejuvaawdssrcrlluewxmxazcaraqbasapmxrypxqqbzwmioackdbwdctrrofisxiecmdyzrbpgirfdclfjtfsfylbpyrpnyiaahqsyukyzbixajxfmtutluixsbheqxfeedwbfsauwbqjqkwyvtugmoxevznylodvpudpldqnunhohqnnztgtjvizqkstljsyxcwejdehxjmvi
6
Quit
