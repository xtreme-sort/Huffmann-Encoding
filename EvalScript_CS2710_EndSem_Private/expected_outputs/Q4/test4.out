l 0000
n 0001
o 0010
p 0011
q 0100
x 0101
a 01100
b 01101
c 01110
d 01111
e 10000
f 10001
g 10010
h 10011
i 10100
j 10101
k 10110
m 10111
r 11000
s 11001
t 11010
u 11011
v 11100
w 11101
y 11110
z 11111
cost: 4192
